docker/bootstrap/Dockerfile.percona57

```diff
@@ -11,7 +11,7 @@ RUN apt-key adv --keyserver ha.pool.sks-keyservers.net \
     } | debconf-set-selections && \
     apt-get update && \
     apt-get install -y --no-install-recommends \
-        percona-server-server-5.7 libperconaserverclient18.1-dev && \
+        percona-server-server-5.7=5.7.14* libperconaserverclient18.1-dev && \
```

Comment:
Okay, now I get it:

When the `RUN` command does not change, Docker does not rebuild the image for that particular command. That makes sense. If I understand it correctly, your change would have worked once because the changed `RUN` command triggers a rebuild. When a newer version comes out, you would have to change the `RUN` command again.

Given that, I agree that `--no-cache` is the way to go here. In fact, our script `docker/bootstrap/build.sh` uses that option as well. And that explains why the images I rebuilt today have the latest version.